### PR TITLE
Dmendoza2017 blue mountain

### LINKED ISSUE #10: dmendoza2017's checklist
# My checklist

- [x] Create branch
- [ ] Work Locally
- [ ] Create Commit(s)
- [ ] Create Pull Request
- [ ]  Collaborate
- [ ] Merge!

### PR DESCRIPTION
**Pull request by dmendoza2017**
Creating a list of places to eat in Blue Mountain, On.

Closes #10

### CODE REVIEW

**Comment by G-Wen** [comment]
Do you recommend KFC or Popeyes?

File: dmendoza2017-blue-mountain.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Places to Eat
+
+- The Keg
+- Milestone
+- McDonald's
```
Comment (G-Wen) [inline dmendoza2017-blue-mountain.md:5]: Have you tried the Big Mac with :bacon:?

File: dmendoza2017-blue-mountain.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Places to Eat
+
+- The Keg
```
Comment (tukaidea) [inline dmendoza2017-blue-mountain.md:3]: see you tonight!

File: dmendoza2017-blue-mountain.md (hunk 1 above)
Comment (wcchen2017) [inline dmendoza2017-blue-mountain.md:5]: My fav place. What's new on their menu?

**Review by wcchen2017 — COMMENTED** [review]
Just a comment on your file

File: dmendoza2017-blue-mountain.md (hunk 1 above)
Comment (dmendoza2017) [reply dmendoza2017-blue-mountain.md:5]: Yes, big fan of a Big Mac with bacon :)

**Pull request merged**